tests/test_modes.py

```diff
@@ -91,6 +91,15 @@ def test_should_return_command_that_list_all_changed_files(self):
             "master",
         ]
 
+    def test_should_return_command_that_list_all_changed_files_for_different_branch(
```

Comment:
Nice one but I'm missing a test in `test_pytest_picked.py` too.

Reply:
Done!